frontend/src/components/MainFeatures/__snapshots__/MainFeatures.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`expect to render Main Features component 1`] = `ShallowWrapper {}`;
```

Comment:
@marcianszlovak this doesn't look good. here should be an example of the generated html:
https://jestjs.io/docs/en/snapshot-testing#snapshot-testing-with-jest

Reply:
I rewrote the test and updated the snapshot, is no longer empty